### PR TITLE
Develop to merge on Master: Full Metricks_Tracker App

### PR DESCRIPTION
**Pull request by ClaudiaRojasSoto**
# Description:
This pull request is to merge on the `Master` branch the `develop` branch of Metricks_Tracker, a mobile web application that allows users to check a list of the population of the world through the use of React and Redux. The application is designed to fetch numeric data from an API related to the population of the world and then present it in an organized and user-friendly manner. The project follows best practices for HTML, CSS, and JavaScript, ensuring a high standard of code quality.

## Key Features:

- [x] **Filterable Metrics**
The application's homepage presents a list of categories containing various metrics. Users can easily filter the metrics based on certain parameters. For instance, if users want to see metrics related to specific countries, they can enter the country name in a search field, and the application will filter the metrics accordingly.
- [x] **API Integration**:
The project involves integrating with an external API that provides numeric data based on parameters. This API is open, and the application has been configured accordingly to fetch data securely.
- [x] **Interactive User Interface**:
The user interface of the web app is designed to be engaging and interactive. Numeric values related to each metric category are displayed prominently, making it easy for users to access relevant information quickly. Users can click or tap on a category to navigate to the details page.
- [x] **Details Page for In-Depth Information**:
Upon selecting a specific category from the homepage, users are taken to a details page. Here, the web app fetches additional data from the API to present more in-depth information about the selected category. For example, if the user chooses a specific continent, the details page will display relevant data, such as the number of views for different countries in that continent.
- [x] **Design Guidelines**:
The application adheres to design guidelines that include carefully selected colors, typography, and layout. The overall design concept is attributed to Nelson Sakwa on Behance, and the appropriate credit is given in the README file, in compliance with the Creative Commons license.
- [x] **Testing for Code Reliability**:
To ensure the reliability of the application, unit tests have been implemented for pure functions in the Redux code. Additionally, integration tests using React Testing Library have been created to verify the application's functionality under different scenarios. Mocking techniques have been employed to prevent actual API requests and Redux Store connections during testing.
- [x] **Single-Page Application (SPA) Architecture**:
The web app is built using React and Redux, adhering to the SPA architecture. The data retrieved from the API is stored in the Redux store, which allows for efficient state management and seamless navigation between different pages.
- [x] **Deployment and Online Access**:
The project is successfully deployed and can be accessed online using Render. Users can visit the online version of the app by following the link provided in the README file.
- [x] **Comprehensive Documentation**:
A video presentation has been prepared to showcase the features and functionality of the web app and added to the description of the app on the readme file. The video, limited to 5 minutes, covers a project description, a demonstration of the application's capabilities, and a spotlight on a particularly interesting piece of code or achievement.

I look forward to your feedback and/or approval to proceed with the merge.

Thank you for your time and consideration,
Greetings.
@ClaudiaRojasSoto

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+:root {
+  --font-size-nav: 25px;
+  --background-color: #ec4c8a;
+  --background-color-variant: #b03968;
+  --white: #fff;
+}
+
+body {
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
```
Comment (DuaneDave) [inline src/index.css]: Please use the specified font family in the given template which is Lato and gill sans in order for your design to match the given template

**Review by DuaneDave — CHANGES_REQUESTED** [review]
Hi @ClaudiaRojasSoto 👋, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!



## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you consider them as they can improve your code._




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/index.css (hunk 1 above)
Comment (ClaudiaRojasSoto) [reply src/index.css]: You're right! I forgot That. It's done. Thank you @DuaneDave

Comment (Whoistolu) [reply src/index.css]: -  Well resolved

Comment (ClaudiaRojasSoto) [reply src/index.css]: @DuaneDave 
The part of the code that I made proud it's on the video added to my readme file. It's on the last part of the video and talks about the reload page on the details page.
These lines are in my readme on the description field:
[Here](https://www.loom.com/share/148a4ded8986457e99c918d9a938794c?sid=837952a4-36f1-4d22-aac3-a9264feb2c0b) is the link to the video presentation

**Review by Whoistolu — APPROVED** [review]
### Hi @ClaudiaRojasSoto 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

- [ ]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

- [x] [OPTIONAL] 
1. For group capstone projects the correct flow for requesting reviews is Github flow which means dev merging to main.
2. For solo capstone projects the flow for requesting reviews is Gitflow which means feature branch merging to dev

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.css (hunk 1 above)
Comment (Whoistolu) [reply src/index.css]: I watched the video. Great job done @ClaudiaRojasSoto 🥇

**Pull request merged**